pyRpc/client.py

```diff
@@ -163,7 +163,7 @@ def call(self, method, callback=None, async=False, args=[], kwargs={}):
             self._main_sender = self._context.socket(zmq.REQ)
             self._main_sender.connect(self._address)  
                       
-        self._main_sender.send_pyobj(req)
+        self._main_sender.send_pyobj(req, protocol=PyVersion)
```

Comment:
I don't think there is any connection between the major version of Python, and the the Pickle protocol version. It just so happens that there is a 2 and a 3 right now. There is also 0 which is the ascii format, and 4 which was introduced in python 3.4. But beyond that... this would still be broken if you had a server running python3 and a client running python2. They would speak different protocols because of the python version. It is safer to just use protocol 2 across the board (I think).